pom.xml

```diff
@@ -65,1442 +65,1442 @@
+  </dependency>
+<!-- Elasticsearch end -->
+
+<!-- Git begin -->
+  <dependency>
+    <groupId>org.eclipse.jgit</groupId>
+    <artifactId>org.eclipse.jgit</artifactId>
+    <version>5.4.0.201906121030-r</version>
+    <scope>provided</scope>
+  </dependency>
+<!-- Git end -->
+
+<!-- GoogleCloud begin -->
+  <dependency>
+    <groupId>com.google.cloud</groupId>
+    <artifactId>google-cloud-vision</artifactId>
+    <version>1.14.0</version>
+    <scope>provided</scope>
+  </dependency>
+<!-- GoogleCloud end -->
+
+<!-- GoogleSearch begin -->
+  <dependency>
+    <groupId>org.jsoup</groupId>
+    <artifactId>jsoup</artifactId>
+    <version>1.8.3</version>
+    <scope>provided</scope>
+  </dependency>
+<!-- GoogleSearch end -->
+
+<!-- HtmlFilter begin -->
+<!-- Duplicate entry for org.jsoup-jsoup-1.8.3 skipping -->
+<!-- Duplicate entry for org.apache.commons-commons-lang3-3.3.2 skipping -->
+<!-- HtmlFilter end -->
+
+<!-- HtmlParser begin -->
+<!-- Duplicate entry for org.jsoup-jsoup-1.8.3 skipping -->
+<!-- HtmlParser end -->
+
+<!-- ImapEmailConnector begin -->
+  <dependency>
+    <groupId>javax.mail</groupId>
+    <artifactId>mail</artifactId>
+    <version>1.4.7</version>
+    <scope>provided</scope>
+  </dependency>
+<!-- ImapEmailConnector end -->
+
+<!-- InMoov begin -->
+  <dependency>
+    <groupId>inmoov.fr</groupId>
+    <artifactId>inmoov</artifactId>
+    <version>1.1.22</version>
+    <scope>provided</scope>
+    <type>zip</type>
+  </dependency>
+  <dependency>
+    <groupId>inmoov.fr</groupId>
+    <artifactId>jm3-model</artifactId>
+    <version>1.0.0</version>
+    <scope>provided</scope>
+    <type>zip</type>
+  </dependency>
+<!-- InMoov end -->
+
+<!-- InMoov2 begin -->
+<!-- skipping fr.inmoov inmoov2 fr.inmoov-inmoov2-null null version/latest -->
+<!-- InMoov2 end -->
+
+<!-- IndianTts begin -->
+  <dependency>
+    <groupId>org.myrobotlab.audio</groupId>
+    <artifactId>voice-effects</artifactId>
+    <version>1.0</version>
+    <scope>provided</scope>
+    <type>zip</type>
+  </dependency>
+<!-- IndianTts end -->
+
+<!-- IntegratedMovement begin -->
+<!-- Duplicate entry for inmoov.fr-inmoov-1.1.22 skipping -->
+<!-- Duplicate entry for inmoov.fr-jm3-model-1.0.0 skipping -->
+<!-- IntegratedMovement end -->
+
+<!-- JFugue begin -->
+  <dependency>
+    <groupId>jfugue</groupId>
+    <artifactId>jfugue</artifactId>
+    <version>5.0.7</version>
+    <scope>provided</scope>
+  </dependency>
+<!-- JFugue end -->
+
+<!-- JMonkeyEngine begin -->
+  <dependency>
+    <groupId>org.jmonkeyengine</groupId>
+    <artifactId>jme3-core</artifactId>
+    <version>3.3.2-stable</version>
+    <scope>provided</scope>
+  </dependency>
+  <dependency>
+    <groupId>org.jmonkeyengine</groupId>
+    <artifactId>jme3-desktop</artifactId>
+    <version>3.3.2-stable</version>
+    <scope>provided</scope>
+  </dependency>
+  <dependency>
+    <groupId>org.jmonkeyengine</groupId>
+    <artifactId>jme3-lwjgl3</artifactId>
```

Comment:
Your putting lwjgl3 back in ?  It will break oculus .. again...

Reply:
merge problem.. will fix. good catch.